src/olympia/amo/templatetags/jinja_helpers.py

```diff
@@ -624,17 +614,11 @@ def nl2br(string):
 
 
 @library.filter(name='datetime')
-def datetime_filter(t, fmt=None):
-    """Call ``datetime.strftime`` with the given format string.
-
-    This is a copy from jingo until we have time to port all our templates
-    to use django's datetime filter.
+def datetime_format(value, format='F j, Y'):
```

Comment:
May as well call it `def format_datetime`, use `format='DATETIME_FORMAT'`.  And I think we're going to need a `def format_date` after-all too.